x-pack/plugins/ml/public/application/app.tsx

```diff
@@ -186,17 +191,17 @@ export const renderApp = (
     lens: deps.lens,
   });
 
+  // Temp
```

Comment:
Will you resolve this as part of this PR? Otherwise it might need a more detailed `TODO` comment.